### PR TITLE
Add RN50 data pipeline perf test

### PR DESCRIPTION
**Pull request by JanuszL**
- adds options to nvJPEG RN50 dataset tests
- adds NR50 pipeline perf test

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

### CODE REVIEW

**Comment by JanuszL** [comment]
Builds 635798

**Comment by JanuszL** [comment]
Rebuilds  635812

File: dali/test/python/test_RN50_data_pipeline.py [hunk 1]
```diff
@@ -0,0 +1,192 @@
+# Copyright (c) 2017-2018, NVIDIA CORPORATION. All rights reserved.
```
Comment (Kh4L) [inline dali/test/python/test_RN50_data_pipeline.py]:
```suggestion
# Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
```

Comment (JanuszL) [reply dali/test/python/test_RN50_data_pipeline.py]: Done

**Comment by ptrendx** [comment]
Can we also have fp16 NHWC tested?

**Comment by JanuszL** [comment]
> Can we also have fp16 NHWC tested?

Done

**Comment by JanuszL** [comment]
Rebuilds 636273

**Pull request merged**